spec/models/manageiq/providers/ovirt/infra_manager/refresher/ovirt_refresher_spec_common.rb

```diff
@@ -39,17 +36,4 @@ def init_connection_vcr(path_to_recording = nil, is_recording: false)
 
     stub_const("OvirtSDK4::Connection", Spec::Support::OvirtSDK::ConnectionVCR)
   end
-
-  def init_inventory_wrapper_class
-    @inventory_wrapper_class = ManageIQ::Providers::Ovirt::InfraManager::Inventory
-
-    allow_any_instance_of(@inventory_wrapper_class).to(receive(:api).and_return("4.2.0_master."))
-    allow_any_instance_of(@inventory_wrapper_class).to(receive(:service)
-                                                         .and_return(OpenStruct.new(:version_string => '4.2.0_master.')))
-  end
```

Comment:
Same here this is just a simple API call that was captured by the response YAMLs